### ISSUE TITLE
Details how to set Keys from the CLI would be helpful

### ISSUE THREAD

**Issue by Frannsoft** [Wyamio/Wyam.Web#61]
I wasn't able to find any info in the docs about setting Keys from the command line using the `--settings` command.  If this indeed does not exist currently I'm happy to add it.  

It's something I ran into recently when looking to set DocsKeys like `Title` and `AssemblyFiles` and Keys like `LinkRoot` from the CLI.

Thanks!

**Comment by daveaglick** [comment]
Good catch. The CLI reference needs to be updated across the board. I was originally just copying and pasting the output from `--help` but that's barely usable and is out of date now anyway because I haven't kept up with it. What I really need to do is rewrite that section with more details on each argument.
